src/routes/controllers/auth.js

```diff
@@ -0,0 +1,25 @@
+import apiResponse from '../../utils/apiResponse';
+
+const { errorResponse, successResponse } = apiResponse;
+/**
+ * This class is responsible for handling
+ * authentication in this project
+ */
+class AuthController {
+  /** Handle application login request
+   *
+   * @param {Object} request
+   * @param {Object} response
+   * @returns {Object} response if request succeedss
+   * @returns {Object} error if request fails
+   */
+  static login(request, response) {
```

Comment:
We should create a middleware that validates users info.

Reply:
I have created it